src/commands/settings/leveling/toggle.ts

```diff
@@ -17,13 +16,14 @@ export default class Toggle {
 
   async run(interaction: ChatInputCommandInteraction) {
     const db = await database();
+    const enabled = await db.table("settings")
+      ?.get(`${interaction.guild.id}.leveling.enabled`)
```

Comment:
```suggestion
    const enabled = await db.table("settings")?
      .get(`${interaction.guild.id}.leveling.enabled`)
```
That character belongs there I think ;)